### PR TITLE
USAGOV-1557 cf logs to new relic

### PR DESCRIPTION
**Pull request by arpage**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1557

## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: scripts/cfevents/space-guids [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export dev=ea450105-895e-4a33-9541-6a0df35ee626
```
Comment (akf) [inline scripts/cfevents/space-guids]: oof, and should there be a `dr` here as well?

**Review by akf — COMMENTED** [review]
I haven't tested this, but it looks reasonable. It has reminded me that I wanted to ask about a couple of things, which maybe we should just talk through: 

1. You're doing some setup in .profile and then you run `source ~/.profile` in the scripts for the actual tasks. Jacob and I found this a little confusing, as he started from the assumption that .profile had done the setup and didn't realize that he needed to copy that in elsewhere. Since .profile doesn't automatically run for the tasks, I think it would be less confusing to put this setup in a differently-named "include" file. I could be wrong. 
2. cf roles! I would like the event capturer to run as a service user with OrgAuditor and SpaceAuditor roles, and no Manager or Developer roles, if possible. I realize that the task runner probably needs SpaceDeveloper in order to run tasks.

File: scripts/cfevents/space-guids (hunk 1 above)
Comment (akf) [reply scripts/cfevents/space-guids]: Actually, it would probably be worth using `cf space dev --guid` to gather these guids programatically.

Comment (arpage) [reply scripts/cfevents/space-guids]: file removed.   Single space GUID is now gathered in the script where it is needed.

**Comment by arpage** [comment]
> 1. You're doing some setup in .profile and then you run `source ~/.profile` in the scripts for the actual tasks. Jacob and I found this a little confusing, as he started from the assumption that .profile had done the setup and didn't realize that he needed to copy that in elsewhere. Since .profile doesn't automatically run for the tasks, I think it would be less confusing to put this setup in a differently-named "include" file. I could be wrong.

The profile isn't read unless cf ssh (or a shell invoked with cf ssh) is run with the "login shell" option.   So, it needs to be sourced manually.   I like the include idea, however it is sort of the same thing - we should discuss.

> 2. cf roles! I would like the event capturer to run as a service user with OrgAuditor and SpaceAuditor roles, and no Manager or Developer roles, if possible. I realize that the task runner probably needs SpaceDeveloper in order to run tasks.

I agree, and will investigate.

**Comment by arpage** [comment]
> > 2. cf roles! I would like the event capturer to run as a service user with OrgAuditor and SpaceAuditor roles, and no Manager or Developer roles, if possible. I realize that the task runner probably needs SpaceDeveloper in order to run tasks.
> 
> I agree, and will investigate.

I've removed space developer and added org/space auditor.  Everything seems to run OK.

**Comment by arpage** [comment]
Added fix for dockerhub auth during cf push

**Pull request merged**